ion/parse_text_simple.go

```diff
@@ -124,11 +124,9 @@ func doClobReplacements(str []byte) []byte {
 		case '\r':
 			// We need to treat "\r\n" as "\n", so skip extra if what comes next is "\n".
```

Comment:
Update the comment since we are also normalizing `\r` to `\n`.

It's strange, this code is like this because there was a test that was failing if I didn't retain `\r`.  *shrug*.

Reply:
Fixed.